webapp/portlet/src/main/webapp/vue-apps/idm-users-management/components/UsersManagementUserFormDrawer.vue

```diff
@@ -236,6 +236,11 @@ export default {
               this.fieldError = error;
               throw new Error(error);
             });
+          } else if (resp.status === 500){
+            return resp.text().then(error => {
+              this.fieldError = error;
+              throw new Error(this.$t('IDMManagement.error.LoginForDeletedUser'));
```

Comment:
This will depend on the returned error, having error HTTP 500 does not mean that the username exists.
Besides checking the Rest service I found that there is a HTTP error 400 returned when the username is already found , could you recheck  what is the real cause of the server error?